composer/datasets/synthetic.py

```diff
@@ -1,5 +1,8 @@
 # Copyright 2021 MosaicML. All Rights Reserved.
 
+import random
```

Comment:
The synthetic datasets file is getting kidna large...perhaps can you make it into a folder, and then have a `synthetic_batch_pair_dataset.py`, `synthetic_pil_image_dataset.py`, `synthetic_bert_dataset.py`, and `synthetic_lm_dataset.py`?